frontend/playwright.config.ts

```diff
@@ -47,6 +48,16 @@ const browserConfigurations: Project[] = [
     },
     dependencies: ["setup"],
   },
+  {
+    name: "mobile",
+    use: {
+      ...devices["Desktop Firefox"],
```

Comment:
Can’t remember what we discussed. We’re using "Desktop Firefox" with a small viewport here instead of an actual mobile device, right? whats the best practice here?  I assume it works but just curious.

Reply:
We can get an actual device, but we wanted to not use the name of a device to mislead people that we are testing on a specific device name, so I made a custom configuration here for our own extending the desktop of firefox and then enabling touch (disabling hover ...etc) and limiting the viewport to certain dimensions

Reply:
Ok yeah i get it. Covering the basics with this custom setup, so does the job we need without misleading with the device name 👍. I wonder if we may miss a few things out of the box when not using a built-in mobile device? Maybe something to look into later, but i also guess this works fine for the current needs.